Helper/AutomatedTesting.php

```diff
@@ -165,6 +214,53 @@ public function __construct(
         $this->stockRegistry = $stockRegistry;
     }
 
+    /**
+     * Retrieve an Order from the repository
+     *
+     * @return Order|null
+     */
+    protected function getPastOrder()
+    {
+        try {
+            $sortOrder = $this->sortOrderBuilder
+                ->setField('entity_id')
+                ->setDirection('DESC')
+                ->create();
+
+            $nonZeroDiscountTaxSearchCriteria = $this->searchCriteriaBuilder
+                ->addFilter('discount_amount', 0, 'lt')
+                ->addFilter('tax_amount', 0, 'gt')
+                ->setPageSize(1)
+                ->setCurrentPage(1)
+                ->setSortOrders([$sortOrder])
+                ->create();
+
+            $defaultSearchCritieria = $this->searchCriteriaBuilder
+                ->setPageSize(1)
+                ->setCurrentPage(1)
+                ->setSortOrders([$sortOrder])
+                ->create();
+
+            $ordersFound = $this->orderRepository
+                ->getList($nonZeroDiscountTaxSearchCriteria)
+                ->getItems();
+
+            if ($ordersFound === null || count($ordersFound) === 0) {
```

Comment:
can this ever be null? wouldn't it always be an empty array? or is it always going to be null? either way pretty sure we only need one of the two conditions

Reply:
cool